Makefile

```diff
@@ -63,7 +63,8 @@ release-major release-minor release-patch:
 
 .PHONY: setup
 setup:
-	$(NPM) install
+	$(YARN)
+	$(YARN) check
```

Comment:
The snippet I suggested earlier specifes that __yarn.lock__ depends only on __package.json__. Unless the modification date of __package.json__ changes, Make will refuse to rebuild __yarn.lock__. To force Make to do your bidding you could run `touch package.json && make yarn.lock` or simply `rm yarn.lock && make yarn.lock`.